vf_pt_BR/dash_core.php

```diff
@@ -547,11 +595,14 @@
 $Definition['Requires: '] = 'Exige: ';
 $Definition['Require users to confirm their email addresses (recommended)'] = 'Exigir que as pessoas confirmem seu e-mail (recomendado)';
 $Definition['Rescan'] = 'Re-escanear';
+$Definition['Reset'] = 'Reset';
 $Definition['Reset my password'] = 'Resetar minha senha';
 $Definition['Reset password and send email notification to user'] = 'Resetar senha e enviar notificação ao usuário';
 $Definition['Resetting the password for %s.'] = 'Resetando a senha de %s.';
 $Definition['Restart'] = 'Recomeçar';
 $Definition['Reveal Password'] = 'Revelar senha';
+$Definition['Revert to Original Version'] = 'Revert to Original Version';
```

Comment:
```suggestion
$Definition['Revert to Original Version'] = 'Reverter Para Versão Original';
```

Reply:
usually, update needs to be done in transifex, If we do it in locales, the next times transifex is synched, it will overwrite it. Translations are done by transifex.